cmd/govim/main_test.go

```diff
@@ -162,13 +156,45 @@ func TestInstallScripts(t *testing.T) {
 
 	govimPath := strings.TrimSpace(runCmd(t, "go", "list", "-m", "-f={{.Dir}}"))
 
+	// For the tests where we set GOVIM_USE_GOPLS_FROM_PATH=true, install
+	// gopls to a temp dir and add that dir to our PATH
+	td, err := installGoplsToTempDir()
+	if err != nil {
+		t.Fatalf("failed to install gopls to temp directory: %v", err)
+	}
+	defer os.RemoveAll(td)
+
 	t.Run("scripts", func(t *testing.T) {
 		testscript.Run(t, testscript.Params{
 			Dir: "testdatainstall",
 			Setup: func(e *testscript.Env) error {
 				e.Vars = append(e.Vars,
 					"PLUGIN_PATH="+govimPath,
-					"CURRENT_GOPATH="+os.Getenv("GOPATH"),
+					"CURRENT_GOPATH="+strings.TrimSpace(runCmd(t, "go", "env", "GOPATH")),
+					testsetup.EnvLoadTestAPI+"=true",
+				)
+				return nil
+			},
+		})
+	})
+
+	t.Run("scripts-with-gopls-from-path", func(t *testing.T) {
+		testscript.Run(t, testscript.Params{
+			Dir: "testdatainstall",
+			Setup: func(e *testscript.Env) error {
+				var currPath string
+				for i := len(e.Vars) - 1; i >= 0; i-- {
+					v := e.Vars[i]
+					if strings.HasPrefix(v, "PATH=") {
+						currPath = v
+						break
+					}
+				}
```

Comment:
Do you care about the case that there is no `PATH` set (i.e., `currPath == ""`)?

Reply:
Nope, that's fine. This is all just defensive in case other code setup changes/relies on `PATH`

Reply:
Well, if `currPath == ""` then you end up with a bit of a malformed PATH (with a trailing separator).

Reply:
TIL via @mvdan that a trailing `:` is valid, but in bash (at least) it means:

> A zero-length (null) directory name in the value of PATH indicates the current directory.

All this time I had assumed it would simply be ignored. 

Fixed, thanks for flagging for attention!